### DISCUSSION TITLE
Inventaire des imprimantes réseaux en SNMP

### DISCUSSION THREAD

**Discussion by babydunet** [glpi-project/glpi-agent#552] [Q&A] [answered]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

None

### Describe the bug

Bonjour à tous,

Je viens de finir l'installation et la configuration de GLPI 10.0.10 sous Debian, j'ai déployé les agents (GLPI AGENT 1.6.1) via GPO sur mon parc informatique et tout fonctionne a la perfection, sauf du coté de l'inventaire SNMP pour les imprimantes sous réseau.

J'ai bien suivi la documentation et les tutoriels sur internet, donc j'ai crée la PLAGE IP de mes imprimantes ainsi que la Tache découverte réseau et Inventaire SNMP, la découverte réseau se passe bien, l'agent utilisé trouve 109 adresse IP dans ma plage IP sauf que la tache SNMP ne donne aucun résultat et affiche ce message "The preparation of the task did not return any targets" contrairement à la découverte réseau la tache SNMP ne se lance même pas alors qu'elle est bien activée.
Est ce que j'ai zappé quelque chose ? Aidez moi svp il ne me reste que ça pour finaliser le projet GLPI.

Merci à tous !

### To reproduce

![c1](https://github.com/glpi-project/glpi-agent/assets/85256228/5e0e8742-842a-4867-89df-30c155db7b84)
![c2](https://github.com/glpi-project/glpi-agent/assets/85256228/bab7e5cd-94e4-4449-89f6-e0cc3f3723fa)
![c3](https://github.com/glpi-project/glpi-agent/assets/85256228/1d787352-5c50-4480-ad71-6bc80f38a547)
![c4](https://github.com/glpi-project/glpi-agent/assets/85256228/6f62b1a2-1840-40da-a76b-9569c7dcb425)

### Expected behavior

Inventaire des imprimantes en SNMP

### Operating system

Linux

### GLPI Agent version

1.6.1

### GLPI version

10.0.10

### GLPIInventory plugin or FusionInventory for GLPI plugin version

GLPI Inventory v1.3.3

**Comment by g-bougard** [comment, marked as answer]
Salut @babydunet 

j'ai déplacé ta question commme "discussion Q&A" car tu n'es clairement pas en train de parler d'un bug et ça ne concerne même probablement pas l'agent GLPI lui-même.

Note que si tu veux de l'aide de la communauté, tu ferais mieux de lancer une discussion en anglais.

Pour ton problème, as-tu vérifié que des credentials SNMP étaient bien associés dans ta plage IP ? A mon avis, il y a un problème ergonomique à la création d'une plage et on peut parfois zapper d'aller dans l'onglet "Identifiants SNMP associés". Si aucun indentifiant n'est associé à la plage, cela peut expliquer que la tâche d'inventaire réseau n'ait rien trouvé à inventorier.

**Reply by babydunet** [reply]
Salut g-bougard et merci pour ta réponse,
Actuellement j'utilise les identifiants "Public community v1 et v2c"qui nécessite aucune identification, vu que j'utilise déjà une solution lexmark qui remonte les imprimantes avec l'inventaire SNMP, par contre pour GLPI il n'y a que le NetDiscover qui fonctionne.

Merci encore une fois

**Comment by steph461** [comment]
@babydunet 

au cas où tu as bien a ajouté ces informations  dans ta plage ip  ou tes plages ip si tu as plusieurs groupes ?
![image](https://github.com/glpi-project/glpi-agent/assets/68054991/3722968d-232f-42de-ab48-4a54adde95d0)
